docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -10,7 +10,7 @@ For the Impatient
 These instructions will create a developer install of Pulp on a dedicated
 development instance.
 
-* Start a Fedora 20 x86_64 instance that will be dedicated for development with
+* Start a RHEL 7 or Fedora 20 x86_64 instance that will be dedicated for development with
```

Comment:
@AndreaGiardini I just hit the same issue :koala:, I believe it's trying to run flake8 and failing to find the binary.

try `yum install -y python-flake8` and then re-run the tests.

Thanks for the feedback on the PR!